### PR TITLE
Change SAML Package to the new namespace

### PR DESCRIPTION
**Pull request by mabumusa1**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
This package https://github.com/lightSAML/lightSAML is no longer in use, this PR switch to https://github.com/litesaml/lightsaml

According to the docs no code change needs to be done

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make sure SSO and saml functions works as expected

### CODE REVIEW

**Review by volha-pivavarchyk — APPROVED** [review]
SSO and saml functions work.

1. I was testing via oktadev.com.
image

2. I was testing via onelogin.com. The xml file loads fine, the redirect works fine too. I haven't logged in because of the settings.

File: composer.json
Comment (kuzmany) [inline composer.json:25]: Should not be part of https://github.com/mautic/mautic/blob/5.x/app/composer.json ?  @escopecz times ago write how to add this https://mautic.slack.com/archives/CQMKV0RU1/p1645619361591259

Comment (mabumusa1) [reply composer.json:25]: @kuzmany you are right, I will adjust this PR. thanks for the note

Comment (mabumusa1) [reply composer.json:25]: @kuzmany @escopecz the package `lightSAML/SymfonyBridgeBundle` did not update its `composer.json` file. and there is an active PR since Jan without no response https://github.com/lightSAML/SymfonyBridgeBundle/pull/35

What we can do is fork their repo and use it as we did with oauth2

Comment (escopecz) [reply composer.json:25]: I guess we'll have to to unblock. Or do you see an alternative path?

Comment (mabumusa1) [reply composer.json:25]: Please create a fork of their repo, I do not have access to do it. I will take care of the changes

Comment (escopecz) [reply composer.json:25]: https://github.com/mautic/lightsaml Ping me if you'll have troubles with write access.

Comment (escopecz) [reply composer.json:25]: Oops, I created a fork of a wrong repo. Deleted now. The fork already exists: https://github.com/mautic/SymfonyBridgeBundle

@fedys is looking into this.

Comment (mabumusa1) [reply composer.json:25]: @escopecz yes, I was the who forked `SymfonyBridgeBundle`, I was about to do this PR but I got into many things until now. 

@fedys if you want to finish this PR, please feel free. if not I will do it

Comment (mabumusa1) [reply composer.json:25]: At some point this bundle will break the upgrade process, I was trying to upgrade to `5.4` to use `rector` but I got this issue, so it is a good time to resolve this issue as well before it happens
``` 
   - symfony/dependency-injection[v4.0.0-BETA1, ..., v4.4.8] require php ^7.1.3 -> your php version (8.0; overridden via config.platform, actual: 8.0.26) does not satisfy that requirement.
    - lightsaml/symfony-bridge 1.3.0 requires symfony/dependency-injection ~2.7|~3.0|~4.0 -> satisfiable by symfony/dependency-injection[v2.7.0-BETA1, ..., 2.8.x-dev, v3.0.0-BETA1, ..., 3.4.x-dev, v4.0.0-BETA1, ..., 4.4.x-dev].
    
```

Comment (fedys) [reply composer.json:25]: @mabumusa1 I will take care of it

**Comment by fedys** [comment]
Closing in favor of https://github.com/mautic/mautic/pull/12181

**Pull request closed without merging**
